docs/proposals/docker-registry-2-4.md

```diff
@@ -0,0 +1,57 @@
+# Support for Docker Registry 2.4
+## Abstract
+
+A proposal to update the docker/distribution to v2.4.0.
+
+## Motivation
+
+Primary goal is to migrate to Schema 2 [1].
+
+## Compatibility
+
+docker/distribution maintains compatibility with old clients that don't know
+how to work with Schema 2.
+
+##### Docker 1.9
+
+When the manifest is pulled by digest or tag with any docker version, a Schema 1 manifest will be returned.
```

Comment:
Here we are talking about pulling by **digest**.

https://github.com/docker/distribution/blob/master/docs/compatibility.md#synopsis